### ISSUE TITLE
Former CEC building Partner

### ISSUE THREAD

**Issue by Keristiena** [uno-isqa-8950/uno-cpi#1424]
We need to account for organizations that were CEC building partners at one point in time. Perhaps  the CEC flag should be connected to academic year. 
We also need to account for this in reporting.

**Comment by Keristiena** [comment]
@robynlloos  MAke sure this is what we agreed on.

**Comment by robynlloos** [comment]
Yes. This is correct. I will add that the designation for CEC building partners is admin only, so we want to make sure this isn't placed into the create a new community partner form.

I will provide a list of partners and the corresponding academic year/semester so that the student can utilize this information to start this.
